pom.xml

```diff
@@ -76,6 +76,7 @@
     <project.build.outputEncoding>UTF-8</project.build.outputEncoding>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <maven.findbugs.failure.strict>true</maven.findbugs.failure.strict>
+    <maven-release-plugin.version>2.5.3</maven-release-plugin.version>
```

Comment:
As in, if it is used once and only once, don't use a property. Only use a property for cases where you have to keep at least two versions in sync *in the same pom*